docker/builder/Dockerfile

```diff
@@ -23,6 +23,8 @@ RUN tar -xzf "sc-${SAUCE_CONNECT_VERSION}-linux.tar.gz"
 RUN ln -s "${SC_BINARY}" /usr/local/bin/sc
 RUN chmod ugo+w /sauce
 
+RUN npm install -g npm@latest
```

Comment:
maybe, but it doesn't much matter right now. until https://github.com/npm/npm/issues/16807 gets fixed, we can't use npm 5 in the pipeline.